File: test/data/cs/test05/ConsoleOutControllee.cs
public class ConsoleOutControllee : IControllee
{
    public void DoAction2()
    {
        Console.WriteLine("Action2");
    }
    public void DoAction21()
    {
        Console.WriteLine("Action21");
    }
    public void DoAction31()
    {
        Console.WriteLine("Action31");
    }
    public void DoAction42()
    {
        Console.WriteLine("Action42");
    }
    public void DoAction421()
    {
        Console.WriteLine("Action421");
    }
    public void DoAction431()
    {
        Console.WriteLine("Action431");
    }
    public void NoTransition(string state, string transition)
    {
        Console.WriteLine($"NoTransition({state}: {transition})");
    }
    public void OverTransition(string transition)
    {
        Console.WriteLine($"OverTransition({transition})");
    }
}

File: test/data/typescript/test05/ConsoleOutControllee.ts
import {IControllee} from "./IControllee"
export class ConsoleOutControllee implements IControllee
{
    public DoAction2(): void
    {
        console.log("Action2");
    }
    public DoAction21(): void
    {
        console.log("Action21");
    }
    public DoAction31(): void
    {
        console.log("Action31");
    }
    public DoAction42(): void
    {
        console.log("Action42");
    }
    public DoAction421(): void
    {
        console.log("Action421");
    }
    public DoAction431(): void
    {
        console.log("Action431");
    }
    public NoTransition(state: string, transition: string): void
    {
        console.log(`NoTransition(${state}: ${transition})`);
    }
    public OverTransition(transition: string): void
    {
        console.log(`OverTransition(${transition})`);
    }
}